funnel/models/project.py

```diff
@@ -191,6 +192,13 @@ class Project(UuidMixin, BaseScopedNameMixin, Model):
         read={'all'},
         datasets={'primary', 'without_parent', 'related'},
     )
+
+    thumbnail_image = with_roles(
```

Comment:
The column needs a type definition, so this should be:

`thumbnail_image: Mapped[bytes | None] = ...`

(Pending column name change.)